tests/monitoring_integration_test.py

```diff
@@ -192,6 +266,43 @@ def testFrontendServiceSloAlertExists(self):
 		found_latency_alert = self.checkForAlertingPolicy('Frontend Service Latency Alert Policy')
```

Comment:
where is this method defined?

Reply:
It's defined at line 255 (or line 181 of the old file)